android/runtime/v8/src/native/V8Util.cpp

```diff
@@ -146,17 +144,41 @@ void V8Util::openJSErrorDialog(Isolate* isolate, TryCatch &tryCatch)
 		return;
 	}
 
+	HandleScope scope(isolate);
+
 	Local<Context> context = isolate->GetCurrentContext();
 	Local<Message> message = tryCatch.Message();
-	Local<Value> stack = tryCatch.Exception().As<Object>()->Get(context, STRING_NEW(isolate, "stack")).ToLocalChecked();
+	Local<Value> exception = tryCatch.Exception();
+
+	Local<Value> jsStack;
+	Local<Value> javaStack;
+
+	if (exception->IsObject()) {
+		Local<Object> error = exception.As<Object>();
+		jsStack = exception.As<Object>()->Get(STRING_NEW(isolate, "stack"));
```

Comment:
use the `error` objet reference you assign just above here rather than call .As<Object> again